From aa7a29e439aca93dfbb7429dbd7ba5fe8de288ca Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Fri, 16 Apr 2021 18:33:44 +0200
Subject: [PATCH] Add missing include of unistd.h

This fixes building on at least Musl, where F_OK is undefined otherwise
---
 src/usb_moded.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/usb_moded.c b/src/usb_moded.c
index e434581..d0b5820 100644
--- a/src/usb_moded.c
+++ b/src/usb_moded.c
@@ -61,6 +61,7 @@
 #endif
 
 #include <getopt.h>
+#include <unistd.h>
 
 #ifdef SAILFISH_ACCESS_CONTROL
 # include <sailfishaccesscontrol.h>
-- 
2.31.1

